_data/mentors.yml

```diff
@@ -1464,3 +1464,36 @@
   network:
   - linkedin: www.linkedin.com/in/samantha-cohen-59a98265
   - medium: https://medium.com/@samanthacohen294
+
+- name: Anahi Gaetan
+  disabled: false
+  matched: false
+  sort: 150
```

Comment:
I think if we want to spotlight her profile because of the ad-hoc availability, we can use 200 for the sort @mads6879 
please correct me if I'm wrong @dricazenck but I think we use the sort values in order of -1, 10, 100, 200, 300, right?

Reply:
yes you are correct, but there are no strict numbers, but I agree she is 5 hours available we could add as 300.